desktop/apps/auction2/components/DOM.js

```diff
@@ -1,18 +1,22 @@
 import ConfirmRegistrationModal from 'desktop/components/credit_card/client/confirm_registration.coffee'
 import PropTypes from 'prop-types'
-import { Component } from 'react'
 import infiniteScroll from 'desktop/apps/auction2/utils/infiniteScroll'
 import mediator from 'desktop/lib/mediator.coffee'
 import scrollToTop from 'desktop/apps/auction2/utils/scrollToTop'
+import { Component } from 'react'
+import { connect } from 'react-redux'
 
-export default class DOM extends Component {
+class DOM extends Component {
   static propTypes = {
     auction: PropTypes.object.isRequired,
     children: PropTypes.node.isRequired,
-    store: PropTypes.object.isRequired,
     user: PropTypes.object
   }
 
+  static contextTypes = {
```

Comment:
I'll pass `context.dispatch` as the alternative (use `mapDispatchToProps` and _only_ pass dispatch) feels like an antipattern compared to the recommended way to get access to the store from within a component tree.